examples/ComplexExample/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "RNExperimental",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "react": "15.4.2",
+    "react-native": "0.40.0",
+    "react-native-vector-icons": "^4.0.0",
+    "react-navigation": "git+https://github.com/react-community/react-navigation.git",
```

Comment:
HI @ericvicenti, I'm having some duplicate module issues when trying to run alongside react-navigation. The error occurs when i run ```npm install``` on the example then try to ```react-native start``` whilst in the react-navigation root dir.

I've tried to mimic the setup of the other examples without any luck. Here is an example of the errors...

```
Failed to build DependencyGraph: @providesModule naming collision:
  Duplicate module name: react-native-vector-icons
  Paths: /Users/simonharper/www/react-navigation/examples/ComplexExample/node_modules/react-native/local-cli/core/__fixtures__/files/package.json collides with /Users/simonharper/www/react-navigation/node_modules/react-native/local-cli/core/__fixtures__/files/package.json

This error is caused by a @providesModule declaration with the same name across two different files.
Error: @providesModule naming collision:
  Duplicate module name: react-native-vector-icons
  Paths: /Users/simonharper/www/react-navigation/examples/ComplexExample/node_modules/react-native/local-cli/core/__fixtures__/files/package.json collides with /Users/simonharper/www/react-navigation/node_modules/react-native/local-cli/core/__fixtures__/files/package.json
```

```
2 info using npm@3.10.3
3 info using node@v6.4.0
```

Originally thought it might be an issue with react-native-vector-icons but failed on other packages when i removed it.

Removing "react-native" from my package.json solves the duplicate module error but am unable to run the example. I'm sure i've overlooked something basic, any suggestions would be appreciated?